javaagent/src/main/java/org/hypertrace/agent/instrument/HypertraceAgent.java

```diff
@@ -103,4 +108,13 @@ static String toOtelPropagators(List<PropagationFormat> propagationFormats) {
     }
     return argsMap;
   }
+
+  private static boolean isTraceableAgentLoaded() {
+    try {
+      Class.forName("ai.traceable.agent.bootstrap.AgentContext", true, null);
```

Comment:
this uses bootstrap classloader and this class seems to be there from the beginning. cc) @surajpuvvada